cfgov/v1/management/commands/report_raw_html.py

```diff
@@ -0,0 +1,142 @@
+import collections
+import csv
+import logging
+import re
+
+from django.apps import apps
+from django.core.management.base import BaseCommand
+
+from wagtail.fields import StreamField
+from wagtail.models import Site
+
+from v1.models import CFGOVPage
+
+
+logger = logging.getLogger(__name__)
+
+
+def find_pattern(data, pattern, key, path=None):
+    """Return the path and match for a given regular expression in JSON
+
+    For the given JSON-like data descend through the data structures looking
+    for strings that match the `re.compile()`ed pattern.
+
+    When one is found, the path to the match will be yielded along with any
+    matching strings. If the path goes through a a JSON array, it will include
+    the index in the array, and if it goes through a JSON object, it'll include
+    the a property of the object with the given key."""
```

Comment:
Something missing here? "the a property of the object"

Reply:
Yeah, left over from a previous iteration. I'll rewrite this.

Reply:
Still rewriting this.

Reply:
Rewritten in [649ac82](https://github.com/cfpb/consumerfinance.gov/pull/7981/commits/649ac82f8b65df6c70787f858441aa7f014fbfa4), hopefully with more clarity!